docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -28,132 +28,91 @@ replacing the properties as appropriate:
+        name.
+      - ``default``
+    * - ``elasticsearch.scroll-size``
+      - Sets the maximum number of hits that can be returned with each
+        Elasticsearch scroll request.
+      - ``1000``
+    * - ``elasticsearch.scroll-timeout``
+      - Amount of time Elasticsearch keeps the
+        `search context <https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html#scroll-search-context>`_
+        alive for scroll requests.
+      - ``1m``
+    * - ``elasticsearch.request-timeout``
+      - Timeout value for all Elasticsearch requests.
+      - ``10s``
+    * - ``elasticsearch.connect-timeout``
+      - Timeout value for all Elasticsearch connection attempts.
+      - ``1s``
+    * - ``elasticsearch.backoff-init-delay``
+      - The minimum duration between backpressure retry attempts for a single
+        request to Elasticsearch. Setting it too low might overwhelm an already
+        struggling ES cluster.
+      - ``500ms``
+    * - ``elasticsearch.backoff-max-delay``
+      - The maximum duration between backpressure retry attempts for a single
+        request to Elasticsearch.
+      - ``20s``
+    * - ``elasticsearch.max-retry-time``
+      - The maximum duration across all retry attempts for a single request to
+        Elasticsearch.
+      - ``20s``
+    * - ``elasticsearch.node-refresh-interval``
+      - How often the list of available Elasticsearch nodes is refreshed.
+      - ``1m``
+    * - ``elasticsearch.ignore-publish-address``
+      - Disables using the address published by Elasticsearch to connect for
+        queries.
+      -
 
 TLS security
 ------------
 
-The Elasticsearch connector provides additional security options to support Elasticsearch clusters that have been configured to use TLS.
-
-The connector supports key stores and trust stores in PEM or Java Key Store (JKS) format. The allowed configuration values are:
-
-``elasticsearch.tls.keystore-path``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The path to the PEM or JKS key store. This file must be readable by the operating system user running Trino.
-
-This property is optional.
-
-``elasticsearch.tls.truststore-path``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The path to PEM or JKS trust store. This file must be readable by the operating system user running Trino.
-
-This property is optional.
-
-``elasticsearch.tls.keystore-password``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The key password for the key store specified by ``elasticsearch.tls.keystore-path``.
-
-This property is optional.
-
-``elasticsearch.tls.truststore-password``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The key password for the trust store specified by ``elasticsearch.tls.truststore-path``.
-
-This property is optional.
+The Elasticsearch connector provides additional security options to support
+Elasticsearch clusters that have been configured to use TLS.
+
+The connector supports key stores and trust stores in PEM or Java Key Store
```

Comment:
we need to reword this and explain that the store files are redundant .. if you have a globally trusted cert on the Elasticsearch cluster it will just work after enabling tls 

The other properties are just if you messed around with self signed certs or custom CAs.